src/System.Collections/src/System/Collections/Generic/SortedSet.cs

```diff
@@ -1142,7 +1107,7 @@ internal virtual void IntersectWithEnumerable(IEnumerable<T> other)
             Clear();
             foreach (T item in toSave)
             {
-                AddIfNotPresent(item);
+                Add(item);
```

Comment:
Why is this a valid change?  This can't be a TreeSubSet that overrides AddIfNotPresent?

Reply:
Nevermind, Add just wraps AddIfNotPresent.